pkgs/by-name/hp/hplip/package.nix

```diff
@@ -161,8 +161,11 @@ python3Packages.buildPythonApplication {
   postPatch = ''
     # https://github.com/NixOS/nixpkgs/issues/44230
     substituteInPlace createPPD.sh \
-      --replace ppdc "${cups}/bin/ppdc" \
-      --replace "gzip -c" "gzip -cn"
+      --replace-fail ppdc "${cups}/bin/ppdc" \
+      --replace-fail "gzip -c" "gzip -cn"
+
+    substituteInPlace Makefile.am \
```

Comment:
Instead of substituting the environment variable in the source, it should be added to `makeFlags` below.